launch/tier4_perception_launch/launch/object_recognition/detection/detector/lidar_dnn_detector.launch.xml

```diff
@@ -18,8 +16,9 @@
         <arg name="output/objects" value="objects"/>
         <arg name="model_name" value="$(var centerpoint_model_name)"/>
         <arg name="model_path" value="$(var centerpoint_model_path)"/>
-        <arg name="model_param_path" value="$(var lidar_model_param_path)/$(var centerpoint_model_name).param.yaml"/>
-        <arg name="class_remapper_param_path" value="$(var lidar_model_param_path)/detection_class_remapper.param.yaml"/>
+        <arg name="model_param_path" value="$(var lidar_model_param_path)/centerpoint.param.yaml"/>
+        <arg name="ml_package_param_path" value="$(var model_path)/$(var centerpoint_model_name)_ml_package.param.yaml"/>
```

Comment:
`model_path` is used but defined in the launcher

Reply:
I fixed at https://github.com/autowarefoundation/autoware.universe/pull/6591/commits/12890c5af469124381727faa677047e24cd2d655